setup.py

```diff
@@ -51,7 +50,7 @@ def get_tests_require() -> List[str]:
 def get_extras_require() -> Dict[str, List[str]]:
 
     requirements = {
-        "checking": ["black", "hacking", "mypy"],
+        "checking": ["black", "flake8", "isort", "mypy"],
```

Comment:
Replaced `hacking` with `flake8`.

Reply:
maybe we should pin `isort` because the release cycle of `isort` is really fast recently.

https://pypi.org/project/isort/#history